src/main/kotlin/mate/academy/Main.kt

```diff
@@ -2,6 +2,10 @@ package mate.academy
 
 fun greetUsers() {
     // implement function
+    print("Please enter a name (or press Enter to quit): ")
+    val name = readLine()
+    println("Hello, $name!")
```

Comment:
If the user presses Enter without typing a name, `readLine()` will return `null`. You should add a check to handle this case, such as prompting the user again or exiting the loop.